src/components/Header/Header.tsx

```diff
@@ -2,8 +2,8 @@ import { useState } from "react";
 import Button from '@material-ui/core/Button';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
-import SunflowerLogo from "../img/sunflowerlogo.png";
-import {ReactComponent as Hamburger} from "../img/hamburger.svg";
+import SunflowerLogo from "../../img/sunflowerlogo.png";
+import { ReactComponent as Hamburger } from "../../img/hamburger.svg";
 import "./Header.css";
```

Comment:
could you add the images into a assets directory:
`components/Header/assets/**.svg`
because the images are only used in the header component